### ISSUE TITLE
Update Grafana v5.4.2 to v7.4.5

### ISSUE THREAD

**Issue by ghost** [bitcoinops/btc-dashboard#6]
I was curious why is an old version of Grafana used for https://dashboard.bitcoinops.org/ ?

![image](https://user-images.githubusercontent.com/13405205/112025391-1ecd7c00-8b5b-11eb-8cd3-55a51be6a30a.png)

Reason: Grafana v5.4.2 had few vulnerabilities including one described in https://rhynorater.github.io/CVE-2020-13379-Write-Up. I had tried testing if it works however got this [image](https://i.imgur.com/ZzZQaEu.png) for everything I tried which confirmed target being vulnerable but cannot be exploited using this CVE because Open Redirect in Gravatar is broken. Also confirmed by Justin Gardner who found this vulnerability in Grafana last year.

**Comment by jnewbery** [comment]
Thanks for raising this, @prayank23!

@bitschmidty, do you mind looking at this?

**Comment by bitschmidty** [comment]
@prayank23 The original developer for the Optech dashboard is no longer working on it. We decided internally to not allocate much time/effort to enhancing it at this time. That is why we have kept it largely as is an not upgraded the Grafana version.

From what Im seeing in your link and description, it seems our instance is a vulnerable version but not currently exploitable due to Gravatar's setup?
